src/Card/Card.tsx

```diff
@@ -14,20 +16,28 @@ const DEFAULT_CARD_HEIGHT = 130;
 
 const StyledCard = styled(MuiCard, {
   shouldForwardProp: (prop) => prop !== 'elevation' && prop !== 'fullWidth',
-})<{ isOver: boolean; fullWidth?: boolean; elevation?: boolean }>(
-  ({ theme, elevation, fullWidth, isOver }) => ({
-    borderRadius: theme.spacing(1),
-    boxShadow: elevation ? theme.shadows[2] : '0px 2px 2px #eeeeee',
-    width: fullWidth ? '100%' : 'max-content',
-    maxWidth: '100%',
-    border: isOver ? '2px solid black' : 'none',
-  }),
-);
+})<{
+  isOver: boolean;
+  fullWidth?: boolean;
+  elevation?: boolean;
+  isSelected?: boolean;
+}>(({ theme, elevation, fullWidth, isOver, isSelected }) => ({
+  borderRadius: theme.spacing(1),
+  boxShadow: elevation ? theme.shadows[2] : '0px 2px 2px #eeeeee',
+  width: fullWidth ? '100%' : 'max-content',
+  maxWidth: '100%',
+  border: isOver
+    ? '2px solid black'
+    : isSelected
+      ? `2px solid ${PRIMARY_COLOR}`
+      : 'none',
+}));
 
 type CardProps = {
   name: string | JSX.Element;
   alt: string;
   id?: string;
+  className?: string;
```

Comment:
Why the use of `className` ?

Reply:
I needed a generic way to select all cards, not based on the id https://github.com/graasp/graasp-builder/blob/f7cc3a22b17a8bbf4b587ebd93b1eb34243cc221/src/components/main/list/SelectionContext.tsx#L95